core/src/main/java/com/netease/arctic/io/writer/SortedPosDeleteWriter.java

```diff
@@ -80,6 +80,17 @@ public SortedPosDeleteWriter(FileAppenderFactory<T> appenderFactory,
     this(appenderFactory, fileFactory, format, mask, index, partitionKey, DEFAULT_RECORDS_NUM_THRESHOLD);
   }
 
+  public SortedPosDeleteWriter(FileAppenderFactory<T> appenderFactory,
+                               OutputFileFactory fileFactory,
+                               FileFormat format,
+                               StructLike partitionKey) {
+    this.appenderFactory = appenderFactory;
```

Comment:
How about use `this(appenderFactory, fileFactory, format, 0, 0, partitionKey, DEFAULT_RECORDS_NUM_THRESHOLD);` instead.

Reply:
Using this constructor, the generated POS-delete file has the wrong name

Reply:
Okay, but I donnot think using a `NULL` data node is a good policy. We could improve this in another PR.